ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml

```
@@ -1,4 +1,92 @@
 periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 25 10 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: pit-7.4.x
+    org: jboss-eap-qe
+    repo: openshift-eap-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: eap-ocp4.15-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-interop-aws
```

```suggestion
  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-interop-aws
  reporter_config:
    slack:
      channel: '#eap-ocp-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :success: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :dumpster-fire:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'
```

I know this is still in draft, but it's easy to forget to add this.